### ISSUE TITLE
LDAP auth response DEBUG request

### ISSUE THREAD

**Issue by sebthebert** [Octopussy-Project/Octopussy#541]
Within this support request I refer to feature request #95 (Octopussy LDAP implementation roles/perms). I think that I managed to implement all the code of revision 592 successfully.

Nonetheless the else part (line 31) due to "defined $auth->{login}" (line 12) in /usr/share/octopussy/login.asp is executed with message $Session->{AAT_MSG_ERROR} = "_MSG_INVALID_LOGIN_PASSWORD"; (line 33) probably since "Checks User/Password from LDAP" (line 109) in /usr/share/perl5/AAT/LDAP.pm returns a wrong result. In short tghe LDAP auth fails.

My coniguration data includes the things defined in feature request #95 plus bind dn and bind pw. The restriction however is set to uidNumber=60076 and that LDAP user is listed successfully within the users manager (user.asp). After I changed altered the account (user_edit.asp?user=mathias.hollstein) it appeared within /srv/lib/octopussy/conf/users.xml as type LDAP with password="". Is this the anticipated behaviour?

further I set default roles within User.pm and LDAP.pm to "ro" and state "Disabled" by default. This however did not change existing accounts (which seems ok).

In short I ask you to provide me with some hints or perl code (since I'm not a perl programme) to debug the LDAP server response or (in case you think that is not the problem) to debug other sections. A final word: I have time until Thue, 10.01.2013 to resolve this. After that deadline my employer will withdraw me from the monitoring project altogether. I'm sorry for the shitty news! :-(
